src/javis_viewer.jl

```diff
@@ -236,3 +235,16 @@ function _javis_viewer(
     end
 
 end
+
+"""
+_jupyter_viewer(video::Video, frames::Int, actions::Vector)
+
+Creates an interactive viewer in a Jupyter Notebook.
+"""
+function _jupyter_viewer(video::Video, frames::Int, objects::Vector)
+    t = Interact.textbox(1:frames, value = 1, typ = "Frame")
+    f = Interact.slider(1:frames, label = "Frame", value = t)
+    output = @map get_javis_frame(video, objects, &f)
```

Comment:
```suggestion
    obs = Observables.async_latest(f)
    output = @map get_javis_frame(video, objects, &obs)
```

You can try something like the above, where `Observables.async_latest(f)` discards past updates if updates come faster than the rendering speed. Also `Observables.throttle(dt, f)` could be a reasonable approach here.

Reply:
Ah. Cool.. Thanks a lot! :)